script/run-comp-match

```diff
@@ -32,12 +32,14 @@ def construct_match_data(args: argparse.Namespace) -> controller_utils.MatchData
         for tla in args.tla
     ]
 
-    recording_config = None  # use the default
+    recording_config = controller_utils.RecordingConfig(
+        resolution=controller_utils.Resolution(args.resolution[0], args.resolution[1]),
+        quality=100,
```

Comment:
Maybe just make this `0 if args.no_record else 100`, and save the mutation below?

Reply:
28f292460e8219448d57b8c5236ea0c02fafa175